controllers/members.js

```diff
@@ -11,7 +13,7 @@ module.exports = (app) => {
 
     const memberList = await metis.getMember({
       channel: tableData.account,
-      account: req.user.record.account,
+      account: req.device.type === 'phone' ? req.headers.channelpublic : req.user.record.account,
```

Comment:
also, lets start making the code more robust by using enumerations instead of passing plain strings.   https://www.sohamkamani.com/javascript/enums/